Main1.cpp

```diff
@@ -17,15 +18,22 @@ int main()
     cout << "Enter a Shape" << endl;
     cin >> shape;
 
-    Draw* shapeDrawer = ShapeFactory :: selectShape(shape);
+    Shape* shapeDrawer = ShapeFactory :: selectShape(shape);
 
+    int area = shapeDrawer -> inputAndCalculateArea();
```

Comment:
Why are we storing the calculated area in a variable when we are not even using it anywhere?